data/categories.txt

```diff
@@ -7668,7 +7675,7 @@ uk:^Шлагбаум
 vi:^Thanh Chắn
 zh-Hans:^水栅门
 zh-Hant:^遮斷機
-el:^Πόρτα φόρτωσης
+el:^Πόρτα ανελκυστήρα
```

Comment:
Are you sure this is correct?
https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dlift_gate

Reply:
> φόρτωσης = loading
> ανελκυστήρα = lift

Okay, but is this exactly the term that is used for https://en.wikipedia.org/wiki/Boom_barrier ?